README.md

```diff
@@ -17,110 +19,132 @@ gem "talkable"
 
 ``` sh
 rails generate talkable:install
-Your talkable site slug (http://talkable.com): 
-You API token (http://talkable.com/sites/zz/edit):
-Do you want a different site to be used for non-production env? (y/n)
-Your staging site slug:
-Your staging site API token:
-
-
-create config/initializers/talkable.rb
-update app/controllers/application_controller.rb
-
-create app/controllers/talkable_invite.rb
-create app/views/talkable_invite/show.html.erb
-update app/layouts/application.html.erb # floating widget install
-update app/layouts/_talkable_floating_widget.html.erb
-update config/routes.rb
+Your Talkable site slug: spree-example
+Your Talkable API Key: SOME-API-KEY
+Do you have a custom domain? [Y/n] n
+
+      create  config/initializers/talkable.rb
+      insert  app/controllers/application_controller.rb
+      insert  app/controllers/application_controller.rb
+      create  app/views/shared
+      create  app/views/shared/_talkable_offer.html.erb
+      insert  app/views/layouts/application.html.erb
+      create  app/controllers/invite_controller.rb
+      create  app/views/invite/show.html.erb
+       route  get '/invite' => 'invite#show'
 ```
 
 ## Configuration
 
 ``` ruby
-Talkable.configure do |c|
-  # required
-  c.site_slug = 'hello'
-  # or
-  c.site_slug = Rails.env.production? ? "hello" : "hello-staging"
-  # required
-  c.api_token = Rails.env.production? ? "1235" : "6789" 
-  # required
-  c.server = 'http://invite.site.com' # fetched from site settings automatically using generator
-  # optional
-  c.js_integration_library = 'http://d2jj/integration/hello.js'  # default
+Talkable.configure do |config|
+  # site slug takes form ENV["TALKABLE_SITE_SLUG"]
+  config.site_slug  = "spree-example"
+
+  # api key takes from ENV["TALKABLE_API_KEY"]
```

Comment:
`api key is taken from ENV["TALKABLE_API_KEY"]` or `takes api key from ENV["TALKABLE_API_KEY"]`